hedera-node/hedera-app/src/main/java/com/hedera/node/app/version/HederaSoftwareVersion.java

```diff
@@ -77,17 +75,12 @@ public HederaSoftwareVersion(
         this.servicesVersion = Objects.requireNonNull(servicesVersion, "servicesVersion must not be null");
     }
 
-    /**
-     * Constructs a new instance with the given versions. This constructor derives config version from {@code build} parameter of {@code servicesVersion}.
-     * @param hapiVersion HAPI version
-     * @param servicesVersion services version
-     */
-    public HederaSoftwareVersion(final SemanticVersion hapiVersion, @NonNull final SemanticVersion servicesVersion) {
-        this.hapiVersion = hapiVersion;
-        this.servicesVersion = Objects.requireNonNull(servicesVersion, "servicesVersion must not be null");
-        if (!StringUtils.isEmpty(servicesVersion.build())) {
-            this.configVersion = Integer.parseInt(servicesVersion.build());
-        }
+    public SemanticVersion servicesVersion() {
```

Comment:
We should probably add a comment in this class to say that we need to this class to load any state before 0.53. Can be done in later PR.